tests/unit/cluster/cli.tcl

```diff
@@ -54,8 +54,10 @@ start_multiple_servers 3 [list overrides $base_conf] {
     }
 
     test "Verify command got unblocked after resharding" {
-        # this (read) will wait for the node3 to realize the new topology
-        assert_error {*MOVED*} {$node3_rd read}
+        # this (read) will wait for the node3 to realize the new topology and then unblock via MOVED,
+        # or it will unblock with CLUSTERDOWN, depending on the cluster state when clientsCron is called.
+        catch {$node3_rd read} err
+        assert {[string match "*MOVED*" $err] || [string match "*CLUSTERDOWN*" $err]}
```

Comment:
I think the down might be because `cluster-node-timeout` is set to 1 (up on line 13). The new cluster tests set a value of 500ms, the old cluster tests set 3s. So it only takes a slight pause to trigger the failure here. I think updating that to a value of 1000 might stabilize the test a bet.

Reply:
i see that we have a test below that will cover the CLUSTERDOWN case, i am ok with the 1000
```
test "Verify command got unblocked after cluster failure" {
        assert_error {*CLUSTERDOWN*} {$node1_rd read}

        # verify there are no blocked clients
        assert_equal [s 0 blocked_clients]  {0}
        assert_equal [s -1 blocked_clients]  {0}
    }
```